### PR TITLE
chapter 13

### PR DESCRIPTION
**Pull request by daniadore**
ready

### CODE REVIEW

**Comment by daniadore** [comment]
Ready

**Comment by daniadore** [comment]
Ready

File: test/models/micropost_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
```
Comment (anlvq-1058) [inline test/models/micropost_test.rb:2]: delete this file, please

File: test/fixtures/microposts.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```
Comment (anlvq-1058) [inline test/fixtures/microposts.yml:3]: delete this file, please

File: test/controllers/microposts_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
```
Comment (anlvq-1058) [inline test/controllers/microposts_controller_test.rb:2]: delete this file, please

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by daniadore** [comment]
Ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**